app/Console/Commands/SendAcceptanceReminder.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Asset;
+use App\Models\CheckoutAcceptance;
+use App\Models\Setting;
+use App\Models\User;
+use App\Notifications\CheckoutAssetNotification;
+use App\Notifications\CurrentInventory;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Notification;
+
+class SendAcceptanceReminder extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'snipeit:acceptance-reminder';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'This will send users with unaccepted assets a reminder to accept them.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $acceptances = CheckoutAcceptance::pending()->where('checkoutable_type', 'App\Models\Asset')->with(['assignedTo', 'checkoutable.assignedTo', 'checkoutable.model', 'checkoutable.adminuser'])->get();
```

Comment:
Can we make sure we're also skipping archived assets here? (Also the letter casing for `pending()` might or not matter, but I think we generally capitalize it?)